mos_tests/functions/common.py

```diff
@@ -33,14 +32,15 @@ def check_stack_status(stack_name, heat, status, timeout=60):
             :param stack_name: Name of stack
             :param status: Expected stack status
             :param timeout: Timeout for check operation
-            :return True or False
+            :return True if stack status is equals to expected status
+                    False otherwise
     """
     if check_stack(stack_name, heat):
         start_time = time()
         stack_status = [s.stack_status for s in heat.stacks.list()
-                    if s.stack_name == stack_name][0]
-        while stack_status.find('IN_PROGRESS') != -1 and time() < \
-                        start_time + 60 * timeout:
+                        if s.stack_name == stack_name][0]
+        while stack_status.find('IN_PROGRESS') != -1 \
```

Comment:
I got it.
What do you think about whole string for the status; 'CREATE_IN_PROGRESS'?
I tried to figure out list of available statuses, but currently without success...